artio-core/src/main/java/uk/co/real_logic/artio/library/SessionConfiguration.java

```diff
@@ -57,6 +60,7 @@ public static Builder builder()
     private SessionConfiguration(
```

Comment:
If we're only allowing people to use FixT or not based upon classpath then wouldn't it be better to set this on the LibraryConfiguration object rather than the SessionConfiguration object for every session?